include/libtorrent/torrent_status.hpp

```diff
@@ -460,6 +460,7 @@ namespace libtorrent {
 		// was saved.
 		bool need_save_resume = false;
 
+#ifndef TORRENT_NO_DEPRECATE
```

Comment:
where you having second thoughts about making this change to torrent_status? I understand your argument that ``torrent_status`` should be convenient and readable to use, and picking out flags may make that less readable and convenient. In my mind, the main arguments in favor of this change are:

1. making it possible to extend the API with new flags without breaking ABI
2. making the interface to libtorrent smaller, with fewer functions, and possibly simpler to some degree.

At least the ABI argument would apply here